Automattic/jetpack
# Rename Webpack-built files back from `.min.js` and remove hashes

#### Changes proposed in this Pull Request:

We renamed them to `.min.js` to avoid a broken wpcom minifier. But now
that we have `Assets::register_script()` to automatically add
`?minify=false` to them, we can put them back at the `.js` names.

Also we can again remove the hashes from the chunk filenames now that
`@automattic/webpack-rtl-plugin` is fixed.

This should fix #19752, and get us closer on #21343. We've got other
files being processed by gulp instead of webpack that may need attention
(or maybe just conversion to being processed by Webpack) before we can
say #21343 is complete.

#### Jetpack product discussion


PT: p9dueE-3MG-p2
Also #19752 and #21343

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Build output for Connection-UI, JITM, Identity Crisis, the Backup plugin, and Jetpack's admin page shouldn't have changed.
* Does stuff still work? Particularly the Post Editor, the Widget Editor, the Site Editor, Instant Search (and its configurator), and Lazy Images. And Widget Visibility.
* In particular, check Instant Search with an RTL language.
* When testing on wpcom, you may want to remove all the obsolete files left behind by earlier versions of Jetpack to make sure it's not loading one of them.
  * `wp-content/mu-plugins/jetpack/_inc/blocks/[number].[hash].js`
  * `wp-content/mu-plugins/jetpack/_inc/blocks/editor*/.min.*`
  * `wp-content/mu-plugins/jetpack/_inc/blocks/*/view.min.*`
  * `wp-content/mu-plugins/jetpack/_inc/blocks/swiper.[hash].*`
  * `wp-content/mu-plugins/jetpack/_inc/blocks/vendors~swiper.*`
  * `wp-content/mu-plugins/jetpack/_inc/build/widget-visibility/editor/index.min.*`
